config/grafana/dashboards/integration-service-dashboard.json

```diff
@@ -430,6 +430,209 @@
       "timeFrom": null,
       "timeShift": null
     },
+    {
+      "id": 31,
+      "gridPos": {
+        "h": 8,
+        "w": 12,
+        "x": 0,
+        "y": 17
+      },
+      "type": "graph",
+      "title": "#2b Time to Start PipelineRun With Ephemeral Env",
+      "thresholds": [
+        {
+          "$$hashKey": "object:1475",
+          "colorMode": "critical",
+          "fill": true,
+          "line": true,
+          "op": "gt",
+          "value": 5,
```

Comment:
How was this value decided?

Reply:
As you said in your above comments, I updated the wrong graphs here.  I assumed we were just supposed to keep the thresholds the same for the time being.  I'm going to close this PR and open another one.  I'll link the changes I made here to  to STONEINTG-669 so that whomever picks up that ticket doesn't have to create the graphs from scratch.

Reply:
The new PR for the histogram is #423